bindings/pyroot/cppyy/CPyCppyy/include/CPyCppyy/PyException.h

```diff
@@ -39,12 +39,12 @@ namespace CPyCppyy {
 class CPYCPPYY_CLASS_EXTERN PyException : public std::exception {
 public:
     PyException();
-    virtual ~PyException() noexcept;
+    ~PyException() noexcept override;
 
-// give reason for raised exception
-    virtual const char* what() const noexcept;
+    // give reason for raised exception
```

Comment:
```suggestion
// give reason for raised exception
```
We should stick to the upstream code style, which indents the comments one level less (@wlav emphasized that this is important to him)